### DISCUSSION TITLE
Tab character no longer selects completion item

### DISCUSSION THREAD

**Discussion by icholy** [community/community#119480] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Recently, I'm no longer able to select completion items using the tab key.

### Environment

Browser: Mozilla Firefox 124.0.1
OS: Ubuntu 22.04.4 LTS

### Demo

![TabBug](https://github.com/community/community/assets/943597/92fbe1f6-09c0-4255-8b15-0cafc648d5d6)

**Comment by haampie** [comment]
Also hitting <kbd>Enter</kbd> does not work, it just adds a new line and hides the autocompletion box.

There's also an issue when selecting a word in the input field and hitting <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Right arrow</kbd>. Try anywhere else and it selects multiple words per second. Try it in the GitHub input box and it's slow as hell, sometimes halting for a second.

Not sure who thought it was a good idea to make any change to this part of GitHub :confus (oh, autocompletion doesn't work.)

**Reply by icholy** [reply]
I don't think this was an intentional change. It still works as expected in Chrome.

**Comment by icholy** [comment]
This started working again for me.

**Reply by ndamclean** [reply]
Updating to Firefox 125 fixed the issue for me.

This was recommended to me by GitHub support.

**Reply by icholy** [reply]
@ndamclean leave a top level answer with that info and I'll mark it as the answer.

**Comment by ndamclean** [comment, marked as answer]
Updating to Firefox 125 fixed the issue for me.

This was recommended to me by GitHub support.
